test/common.go

```diff
@@ -31,6 +31,8 @@ var sourceArrayUsername = flag.String("sourceArrayUsername", "xxx", "Source arra
 
 var sourceArrayPassword = flag.String("sourceArrayPassword", "xxx", "Source array password")
 
+var pushResultToDB = true
```

Comment:
DB means dashboard?  If so, it needs to be spelled out.

Reply:
changed to meaningful name